rpc-interface/src/types.rs

```diff
@@ -95,16 +93,7 @@ impl FromStr for ValidityStartHeight {
 pub enum HashAlgorithm {
     Blake2b = 1,
     Sha256 = 3,
-}
-
-#[allow(clippy::from_over_into)]
-impl Into<HTLCContractHashAlgorithm> for HashAlgorithm {
-    fn into(self) -> HTLCContractHashAlgorithm {
-        match self {
-            HashAlgorithm::Blake2b => HTLCContractHashAlgorithm::Blake2b,
-            HashAlgorithm::Sha256 => HTLCContractHashAlgorithm::Sha256,
-        }
-    }
+    Sha512 = 4,
```

Comment:
`HashAlgorithm` is unused now?

Reply:
Its actually used by the `rpc-client` [here](https://github.com/nimiq/core-rs-albatross/pull/1668/files#diff-c37ba4db67fea5930db56228ef619010a93f6144fb2d4427c61e8992d5a4346bR10).